### PR TITLE
Add total when the stream ends

### PR DESCRIPTION
**Pull request by Marsup**
For interrupted streams, I'd love to be able to get the total, so I added at the end.

This seems fairly simple and the code already relies on this so I don't know if you want additional tests for it ?

### CODE REVIEW

File: lib/elasticsearch-stream.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ var LibElasticasearchScrollStream = function(client, query_opts, optional_fields
   this._options.scroll = query_opts.scroll || '10m';
   this._reading = false;
   this._counter = 0;
+  this._total = undefined;
```
Comment (francov) [inline lib/elasticsearch-stream.js]: I personally prefer a `0` assignment instead of `undefined`

**Comment by Marsup** [comment]
That's the intention yes. I'll change for a 0.

**Comment by Marsup** [comment]
Added a line of docs, always forget about that :)

**Comment by francov** [comment]
Perfect, thank you!

**Pull request merged**